data/pc/1.20.2/protocol.json

```diff
@@ -117,7 +129,7 @@
                     },
                     {
                       "name": "nbtData",
-                      "type": "optionalNbt"
+                      "type": "optionalNbtRoot"
```

Comment:
Item NBT data is still written with a named root tag.

Reply:
Seems I was actually incorrect here. Just very weird naming in Vanilla codebase that had me confused (i.e. `readUnnamedTag` actually reads a named tag, whereas `readAnyTag` reads an unnamed tag)